tokio/src/macros/cfg.rs

```diff
@@ -183,6 +183,16 @@ macro_rules! cfg_net_unix {
     }
 }
 
+macro_rules! cfg_net_windows {
+    ($($item:item)*) => {
+        $(
+            #[cfg(all(target_os = "windows", feature = "net"))]
+            #[cfg_attr(docsrs, doc(cfg(feature = "net")))]
```

Comment:
We should also mention windows in the `doc(cfg(...))`.

Reply:
```suggestion
            #[cfg_attr(docsrs, doc(cfg(all(target_os = "windows", feature = "net"))))]
```